page_creation.rst

```diff
@@ -133,9 +133,9 @@ variable so we can render that::
         {
             $number = mt_rand(0, 100);
 
-            return $this->render('lucky/number.html.twig', array(
-                'number' => $number
-            ));
+            return $this->render('lucky/number.html.twig', [
+                'number' => $number,
+            ]);
```

Comment:
@bocharsky-bw yes. I think we should wait till PHP <5.4 is completely dropped in Symfony (that'll be when 2.8 reaches end of maintainance). We can then safely do this change for all code examples in all branches at one go.